src/Blocks/BlockObserver.h

```diff
@@ -1,19 +1,42 @@
-
-#pragma once
-
-#include "BlockHandler.h"
-#include "Mixins.h"
-
-
-class cBlockObserverHandler:
-	public cClearMetaOnDrop<cPitchYawRotator<cBlockHandler>>
-{
-	using super = cClearMetaOnDrop<cPitchYawRotator<cBlockHandler>>;
-
-public:
-
-	cBlockObserverHandler(BLOCKTYPE a_BlockType) : super(a_BlockType)
-	{
-	}
-
-};
+
+#pragma once

```

Comment:
I think your git is misconfigured. You are introducing windows style line endings everywhere.

Reply:
sorry 😥